io/private/EptSupport.hpp

```diff
@@ -271,29 +263,29 @@ class PDAL_DLL EptInfo
         Binary
     };
 
-    EptInfo(Json::Value info) : m_info(info)
+    EptInfo(const NL::json& info) : m_info(info)
     {
         m_bounds = toBox3d(m_info["bounds"]);
-        m_points = m_info["points"].asUInt64();
-        m_span = m_info["span"].asUInt64();
-        m_srs = m_info["srs"]["wkt"].asString();
+        m_points = m_info["points"].get<uint64_t>();
+        m_span = m_info["span"].get<uint64_t>();
+        m_srs = m_info["srs"]["wkt"].get<std::string>();
 
         if (m_srs.empty())
```

Comment:
Similar here, maybe this check should be `if (m_srs.empty() && m_info.contains("srs"))`.